aws_xray_sdk/core/utils/sqs_message_helper.py

```diff
@@ -0,0 +1,10 @@
+class SqsMessageHelper:
+    
+    @staticmethod 
+    def isSampled(sqs_message):
+        attributes = sqs_message['attributes']
+
+        if 'AWSTraceHeader' not in attributes:
```

Comment:
I suggest making `AWSTraceHeader` into a constant and reuse it in both the places in this method.